spec/requests/complements_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe 'Complements API' do
+  it 'returns a complement via JSON' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.